features/nationaliteithistorie.feature

```diff
@@ -40,7 +62,63 @@ Functionaliteit: Tonen van Nationaliteithistorie
       Als de nationaliteithistorie met burgerservicenummer 555550004 wordt geraadpleegd
       Dan bevat het nationaliteithistorie-item met nationaliteit.code "0100" het gegeven "datumIngangGeldigheid" met datum "2020-07-27"
 
-
+    # Vervangt bovenstaande scenario
+    # hier is categorie wel toegevoegd om te weten welke datum ingang geldigheid actueel is en welke historisch
+    # vraag: niet duidelijk of het relevant is om de onjuist kolom op te nemen in de historische rij. Voor het bepalen van de gegevens die voor de nationaliteit moet worden geretourneerd lijkt het niet relevant
+    Scenario: persoon heeft een nationaliteit met geactualiseerde datum ingang geldigheid
+      Gegeven een persoon met burgerservicenummer '555550004' heeft een 'nationaliteit' met de volgende gegevens
+      | categorie | nationaliteit (05.10) | datum ingang geldigheid (85.10) |
+      | 4         | 0100                  | 20200727                        |
+      | 54        | 0100                  | 20200713                        |
+      Als nationaliteithistorie wordt geraadpleegd met de volgende parameters
+      | naam                | waarde                                                                   |
+      | type                | RaadpleegMetPeildatum                                                    |
+      | burgerservicenummer | 555550004                                                                |
+      | peildatum           | 2022-08-01                                                               |
+      | fields              | nationaliteiten.nationaliteit.code,nationaliteiten.datumIngangGeldigheid |
+      Dan heeft de response de volgende 'nationaliteiten'
+      | type          | nationaliteit.code | datumIngangGeldigheid.type | datumIngangGeldigheid.datum | datumIngangGeldigheid.langFormaat |
+      | Nationaliteit | 0100               | Datum                      | 2020-07-27                  | 27 juli 2020                      |
```

Comment:
Ik kan mij voorstellen dat de vraag ontstaat of er nu twee voorkomens van de nationaliteit moeten worden getoond als wordt bevraagd met periode van 20200712 tot 20200731.